src/Framework/Sources/JavaScript/Background/chrome/storage.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+// not sure if our local storage is isolated, so better use key prefixing
+function keyName(key) {
+    return '__topee_internal.' + key;
+}
+
+var storage = {
+      /**
+     * @param keys (optional)
+     * @param cb function
+     */
+    get (keys, cb) {
+        const callbackFunc = cb || keys;
+        let keysToFetch = [];
+        let defaults = {};
+        if (Array.isArray(keys)) {
+            keysToFetch = keys;
+        } else if (typeof keys === 'string') {
+            keysToFetch = [keys];
+        } else if (typeof keys === 'object') {
+            keysToFetch = Object.keys(keys);
+            defaults = keys;
+        } else if (typeof keys === 'function') {
+            keysToFetch = Object.keys(localStorage);
```

Comment:
this doesn't go together with keyName

Reply:
yep. there are no tests yet for this case, that's why it went unnoticed

Reply:
fixed